website/docs/r/datadog_tagrules.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor_tagrule"
```

Comment:
I still prefer renaming this to `azurerm_datadog_monitor_tag_rule`.

Reply:
Updated